### DISCUSSION TITLE
Generate Model Information?

### DISCUSSION THREAD

**Discussion by aryant2707** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#19] [Q&A] [answered]
Hi, thank you so much for this! I'm a newbie in the OpenCore world, and I was wondering what you mean by: "The model information has been removed, please generate and replace it yourself"

What and where exactly do I have to generate this information?

**Comment by PIut02** [comment, marked as answer]
Well, that means SystemProductName in the PI - Generic, just click Generate.
